bin/dynamo-tools.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+
+const { tables, tableNames } = require('./dynamo-tables');
+
+class DynamoTools {
+  constructor({ clients, regions }) {
+    this._clients = clients;
+    this._regions = regions;
+  }
+
+  async getTableStatus(region, tableName) {
```

Comment:
JSDoc throughout

Reply:
This is just for tooling / local development stuff so not a very high priority. But sure will add.

Reply:
Right now this is used only to create the tables in localstack.